cylc/flow/network/schema.py

```diff
@@ -1226,36 +1226,50 @@ class Meta:
     depth = Int()
     proxies = graphene.List(
         lambda: FamilyProxy,
-        description='Associated cycle point proxies.',
+        description='Associated family proxy instances.',
         args=PROXY_ARGS,
         strip_null=STRIP_NULL_DEFAULT,
         delta_store=DELTA_STORE_DEFAULT,
         delta_type=DELTA_TYPE_DEFAULT,
         resolver=get_nodes_by_ids)
     parents = graphene.List(
         lambda: Family,
-        description='Family definition parent.',
+        description='Families that this family directly inherits from.',
         args=DEF_ARGS,
         strip_null=STRIP_NULL_DEFAULT,
         delta_store=DELTA_STORE_DEFAULT,
         delta_type=DELTA_TYPE_DEFAULT,
         resolver=get_nodes_by_ids)
     child_tasks = graphene.List(
         Task,
-        description='Descendant definition tasks.',
+        description='Tasks that inherit from this family.',
         args=DEF_ARGS,
         strip_null=STRIP_NULL_DEFAULT,
         delta_store=DELTA_STORE_DEFAULT,
         delta_type=DELTA_TYPE_DEFAULT,
         resolver=get_nodes_by_ids)
     child_families = graphene.List(
         lambda: Family,
-        description='Descendant desc families.',
+        description='Families that inherit from this family.',
         args=DEF_ARGS,
         strip_null=STRIP_NULL_DEFAULT,
         delta_store=DELTA_STORE_DEFAULT,
         delta_type=DELTA_TYPE_DEFAULT,
         resolver=get_nodes_by_ids)
+    descendants = graphene.List(
+        String,
+        description=sstrip('''
+            Linearised first-parent descendants.
+
+            Inheritance in Cylc provides two functions:
```

Comment:
```suggestion
            Inheritance in Cylc fulfils two purposes:
```
(rather minor suggestion, obviously)